packages/showcases/css/stories/components/rating/examples/interactive.html

```diff
@@ -5,65 +5,166 @@
+        type="radio"
+        name="rating1"
+        value="4"
+        id="rating1-4"
+        aria-label="4 stars out of 5"
+      />
+      <label for="rating1-4"></label>
 
-      <input type="radio" id="star1-1" name="rating-1" value="1" />
-      <label for="star1-1" title="1 star out of 5" role="presentation"></label>
+      <input
+        type="radio"
+        name="rating1"
+        value="5"
+        id="rating1-5"
+        aria-label="5 stars out of 5"
+      />
+      <label for="rating1-5"></label>
     </div>
-    <span class="vtmn-rating_comment--primary" aria-label="Article rating"
-      >4.9/5</span
-    >
-    <span class="vtmn-rating_comment--secondary" aria-label="Number of ratings"
-      >(589)</span
-    >
+    <span class="vtmn-rating_comment--primary">4.9/5</span>
+    <span class="vtmn-rating_comment--secondary">(589)</span>
   </div>
 </div>
 
 <div class="block">
-  <div class="vtmn-rating">
+  <div class="vtmn-rating vtmn-rating_size--small">
     <div
       class="vtmn-rating--interactive"
-      aria-label="Rate this article"
+      aria-label="Rate the article"
       role="radiogroup"
     >
-      <input type="radio" id="star5-2" name="rating-2" value="5" />
-      <label
-        for="star5-2"
-        title="5 stars 5 out of 5"
-        role="presentation"
-      ></label>
+      <input
+        type="radio"
+        name="rating2"
+        value="1"
+        id="rating2-1"
+        aria-label="1 star out of 5"
+      />
+      <label for="rating2-1"></label>
 
-      <input type="radio" id="star4-2" name="rating-2" value="4" />
-      <label for="star4-2" title="4 stars out of 5" role="presentation"></label>
+      <input
+        type="radio"
+        name="rating2"
+        value="2"
+        id="rating2-2"
+        aria-label="2 stars out of 5"
+      />
+      <label for="rating2-2"></label>
 
-      <input type="radio" id="star3-2" name="rating-2" value="3" />
-      <label for="star3-2" title="3 stars out of 5" role="presentation"></label>
+      <input
+        type="radio"
+        name="rating2"
+        value="3"
+        id="rating2-3"
+        aria-label="3 stars out of 5"
+      />
+      <label for="rating2-3"></label>
 
-      <input type="radio" id="star2-2" name="rating-2" value="2" />
-      <label for="star2-2" title="2 stars out of 5" role="presentation"></label>
+      <input
+        type="radio"
+        name="rating2"
+        value="4"
+        id="rating2-4"
+        aria-label="4 stars out of 5"
+      />
+      <label for="rating2-4"></label>
 
-      <input type="radio" id="star1-2" name="rating-2" value="1" />
-      <label for="star1-2" title="1 star out of 5" role="presentation"></label>
+      <input
+        type="radio"
+        name="rating2"
+        value="5"
+        id="rating2-5"
+        aria-label="5 stars out of 5"
+      />
+      <label for="rating2-5"></label>
     </div>
-    <span class="vtmn-rating_comment--primary" aria-label="Article rating"
```

Comment:
Why did you delete aria labels for comments?

Reply:
I remobed this aria-label because according MDN : https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Aria-label should be set up when the text is not visible. Here the text is visible so we shouldn't use it.